### ISSUE TITLE
BUG: Using `--source-string` results in error

### ISSUE THREAD

**Issue by tony-griffin** [sensepost/punch-q#4]
When using punch-q from the provided Dockerfile the flag `--source-string` seems to allways throw an error
ie
`punch-q --dump-config --host mq --port 1414 --channel SYSTEM.DEF.SVRCONN --qm_name <QMNAME> --username <USER> --password <PASSWORD> messages push -q '<QUEUENAME>' --source-string 'test'`

This results in the following error:
```python
Message (truncated): test
Traceback (most recent call last):
  File "/usr/local/bin/punch-q", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/libpunchq/cli.py", line 896, in push
    mqqueue.put(message, put_mqmd, put_opts)
  File "/usr/local/lib/python3.7/dist-packages/pymqi/__init__.py", line 1710, in put
    check_not_py3str(msg)  # Python 3 bytes check
  File "/usr/local/lib/python3.7/dist-packages/pymqi/__init__.py", line 144, in check_not_py3str
    raise TypeError("Python 3 style string found but not allowed here: '{0}'. Convert to bytes.".format(var))
TypeError: Python 3 style string found but not allowed here: 'test'. Convert to bytes.
```

### workaround
Currently, we have been able to workaround this by using process substitution with the --source-file cmd instead ie.
`--source-file <(echo 'test')`

**Comment by leonjza** [comment]
Thanks for reporting this! I actually broke punch-q pretty badly with the last tag given how internal data handling changed (with python 3). Part of it is being discussed [here](https://github.com/dsuch/pymqi/issues/152), but I am in the process of working on an interim fix for it which should include fixing his bug.

**Comment by leonjza** [comment]
I just released version 1.3.0 which should fix this.
